src/file-3.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15902/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15902/project/issues?id=sample&pullRequest=feature2&issues=AWG_o9ElChrAhrW64gV2&open=AWG_o9ElChrAhrW64gV2)

[ISSUE_KEY]: AWG_o9ElChrAhrW64gV2